docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,59 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_prepare_model_MO_Python_API}
+
+- Model Optimizer has Python API for model conversion, which is represented by convert_model() method in openvino.tools.mo namespace.
+  convert_model() has all the functionality available from command line tool plus the possibility of passing Python objects (models, extensions) to convert_model() directly from memory.
+  convert_model() returns openvino.runtime.Model object which can be compiled and infered or serialized to IR.
+
+```sh
+from openvino.tools.mo import convert_model
+
+ov_model = convert_model("resnet.onnx")
+```
+
+MO Python API allows conversion of PyTorch models.
+```sh
+import torchvision
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, input_shape=[1,3,100,100])
+```
+
+- Following types are supported as input model for convert_model():
+
+| PyTorch                                                          |                                                                                            TF/ TF2 / Keras                                                                                            |
+|:-----------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| torch.nn.Module torch.jit.ScriptModule torch.jit.ScriptFunction  | tf.keras.Model tf.keras.layers.Layer tf.compat.v1.GraphDef tf.Module tf.compat.v1.wrap_function tf.compat.v1.session tf.train.checkpoint tf.python.training.tracking.base.Trackable(with limitations) | 
+
+- convert_model() accepts all parameters available in MO command line tool. Parameters can be specified by Python classes or string analogs identically to command line tool.
+- Example 1:
+```sh
+from openvino.runtime import PartialShape, Layout
+
+ov_model = convert_model(model, input_shape=PartialShape([1,3,100,100]), mean_values=[127, 127, 127], layout=Layout("NCHW"))
+```
+- Example 2:
+```sh
+ov_model = convert_model(model, input_shape="[1,3,100,100]", mean_values="[127,127,127]", layout="NCHW")
+```
+- Command-line flags like --compress_to_fp16 can be set in Python API by providing a boolean value (True or False).
+```sh
+ov_model = convert_model(model, compress_to_fp16=True)
+```
+- "input" parameter can be set by tuple with name, shape and type. Another option is to use InputCutInfo class which was introduced for complex cases when value also needs to be set.
```

Comment:
is there any discription InputCutInfo in docs? For now, it is not obvious in which cases user needs to provide value and in which format (e.g. is it always list or maybe numpy or tensor)

Reply:
and type of input, also is always numpy?

Reply:
Added more info on InputCutInfo and supported types.